templates/publisher/_privacy-notification.html

```diff
@@ -0,0 +1,12 @@
+<div class="p-strip is-shallow u-no-padding--bottom js-notification-holder">
+  <div class="row">
+    <div class="p-notification--information u-no-margin--bottom">
+      <p class="p-notification__response">
+        <span class="p-notification__status">Information:</span> We have updated our
+        <a href="https://www.ubuntu.com/legal/terms-and-policies/developer-terms-and-conditions" class="p-link--external">developer terms and conditions</a>
+        and <a href="https://www.ubuntu.com/legal/dataprivacy/snap-store" class="p-link--external">privacy notice</a>.
+        <a href="javacsript:;" class="p-notification__action" data-ispermanent="true" data-name="privacy-update-06-2018">Dismiss</a>
```

Comment:
I tried locally it seems it works even without the `href`. Is there a reason why you want to keep it?

Reply:
> `href=""` results in the page jumping to the top/ reloading so I've replaced with `href="javascript:;"` ugly but works.

Inconsitently across browsers.

Reply:
oups sorry haven't seen the comment